codec-mqtt/src/main/java/io/netty/handler/codec/mqtt/MqttUnsubscribeMessage.java

```diff
@@ -22,11 +22,14 @@
  */
 public final class MqttUnsubscribeMessage extends MqttMessage {
 
+    private final MqttUnsubscribePayload payload;
+
     public MqttUnsubscribeMessage(
             MqttFixedHeader mqttFixedHeader,
             MqttMessageIdVariableHeader variableHeader,
             MqttUnsubscribePayload payload) {
-        super(mqttFixedHeader, variableHeader, payload);
+        super(mqttFixedHeader, variableHeader);
+        this.payload = payload;
```

Comment:
There are messages that could have no payload, so this null is for them. However proposed `ObjectUtil.checkNonNull(payload, "payload")` seems unnecessary for me as `MqttDecoder.decodePayload` will always return non-null payload for messages with payload.